example/react-native/src/components/NavigationController.js

```diff
@@ -0,0 +1,14 @@
+import { StackNavigator, DrawerNavigator } from "react-navigation";
+import { HomeScreen, HttpScreen } from "./screens";
+
+/**
+ * Main navigation controller contains all children screen components
+ */
+const NavigationController = DrawerNavigator({
+  Home: StackNavigator({ screen: HomeScreen }),
```

Comment:
Why a `StackNavigator` inside the `DrawerNavigator`. Need the individual screen another navigator?

Reply:
Yes, this puts the Drawer o top of the Toolbar